configurate.go

```diff
@@ -30,8 +30,9 @@ func Init(path string) (*viper.Viper, error) {
 	return cfg, nil
 }
 
-// InitDefaults initializes the underlying config, using defaults for any unspecified configuration settings.
-func InitDefaults(path, defaultConfig string) (*viper.Viper, error) {
+// InitDefaultsReader initializes the underlying config from a file, using defaults for any unspecified configuration
```

Comment:
The comment should start with "InitDefaultsF" to match the name of the function.

Reply:
Ah, thanks for catching that.